scipy/stats/tests/test_distributions.py

```diff
@@ -9249,21 +9249,17 @@ def test_genextreme_give_no_warnings():
         assert_equal(number_of_warnings_thrown, 0)
 
 def test_moments_zero_shape():
-    # Regression test for gh-22532
-    
-    with warnings.catch_warnings(record=True) as w:
```

Comment:
Warnings cause test failures automatically; we don't need to do anything manually.